core/src/avm2/globals/flash/net/url_loader.rs

```diff
@@ -0,0 +1,175 @@
+//! `flash.net.URLLoader` builtin/prototype
+
+use crate::avm2::activation::Activation;
+use crate::avm2::class::Class;
+use crate::avm2::method::{Method, NativeMethodImpl};
+use crate::avm2::names::{Namespace, QName};
+use crate::avm2::object::TObject;
+use crate::avm2::value::Value;
+use crate::avm2::{Error, Object};
+use crate::backend::navigator::RequestOptions;
+use crate::loader::DataFormat;
+use crate::string::AvmString;
+use gc_arena::{GcCell, MutationContext};
+
+/// Implements `flash.net.URLLoader`'s class constructor.
+pub fn class_init<'gc>(
+    _activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    Ok(Value::Undefined)
+}
+
+/// Implements `flash.net.URLLoader`'s instance constructor.
+pub fn instance_init<'gc>(
```

Comment:
Method bindings don't need to be `pub` (some existing code has it unnecessarily)